src/Forcing/simple_forcing.jl

```diff
@@ -1,52 +1,142 @@
 """
-    SimpleForcing{X, Y, Z, F, P}
+    struct SimpleForcing{X, Y, Z, M, F, P}
```

Comment:
Unbelievably minor but I think Julia convention is to not write `struct` in the docstring.

For example:
```
help?> Complex

  Complex{T<:Real} <: Number

  Complex number type with real and imaginary part of type T.
```

Reply:
done!